src/app/accounts/login.component.html

```diff
@@ -44,6 +44,11 @@
                             <i class="fa fa-pencil-square-o" aria-hidden="true"></i> {{'createAccount' | i18n}}
                         </a>
                     </div>
+                    <div class="d-flex">
```

Comment:
Should this be hidden for now from the UI?

Reply:
We can circle back and comment it out if we want to remove for a release.